.travis.yml

```diff
@@ -3,12 +3,24 @@
 # Use of this source code is governed by the Apache License, Version 2.0.
 # See the LICENSE file for details.
 
+# Will upgrade to new container-based infrastructure
 sudo: false
 
+cache:
+  - node_modules
```

Comment:
Tell travis to persist this directory across runs. It holds our required nodejs modules.  Added comment.